EnglishFilter/sections/general_extensions.txt

```diff
@@ -2991,6 +2991,10 @@ uplinks.in,minilinks.in#$##plswait { display: none !important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/71428
 researchgate.net#$#.lite-page__header-navigation--with-ad { top: 0 !important; }
 researchgate.net#$#.research-resources-summary__inner.is-sticky { top: 0 !important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/72091
```

Comment:
I don't see this element.

Reply:
You will have to scroll a bit and/or follow to the next article.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/105983502-e739da80-60a9-11eb-9366-785d47d1120a.png)</details><br/>